src/Groups/GroupDetails/GroupDetails.js

```diff
@@ -137,12 +141,14 @@ GroupDetails.propTypes = {
   editGroup: PropTypes.func.isRequired,
   removeGroup: PropTypes.func.isRequired,
   group: PropTypes.object.isRequired,
+  groupSummary: PropTypes.object,
```

Comment:
```
shape({
...
});
``` 
?

Reply:
mb,don't see many benefits

Reply:
at least other dev can see what structure is expected